packages/auth/e2e/emailLink.e2e.js

```diff
@@ -25,6 +25,13 @@ describe('auth() -> emailLink Provider', () => {
       };
       await firebase.auth().sendSignInLinkToEmail(email, actionCodeSettings);
     });
+
+    xit('should send email with defaults', async () => {
```

Comment:
Guessing this is skipped because you dont know whether an email is sent?

Reply:
I've got this as a broken test from the commit. Will look into why this has been changed...

Reply:
Yeah so on native, iut has success listeners. So I cannot test a response with it set as an email sign in.

```
firebaseAuth
      .createUserWithEmailAndPassword(email, password)
      .addOnSuccessListener(authResult -> {
        Log.d(TAG, "createUserWithEmailAndPassword:onComplete:success");
        promiseWithAuthResult(authResult, promise);
        Log.d(TAG, "createUserWithEmailAndPassword:onComplete:promiseResolved");
      })
      .addOnFailureListener(exception -> {
        Log.e(TAG, "createUserWithEmailAndPassword:onComplete:failure", exception);
        promiseRejectAuthException(promise, exception);
      });
```